### PR TITLE
feat(emails): update copy on account recovery email

### PR DESCRIPTION
**Pull request by chenba**

### CODE REVIEW

**Review by bcolsson — APPROVED** [review]

File: packages/fxa-auth-server/lib/senders/emails/templates/recovery/en.ftl [hunk 1]
```diff
@@ -1,4 +1,8 @@
 recovery-subject = Reset your password
-recovery-title = Need to reset your password?
-recovery-description = Click the button within the next hour to create a new password. The request came from the following device:
+recovery-title-2 = Forgot your password?
+recovery-description-2 = Click the button within the next hour to create a new password.
+# Information on the browser, IP address, date and time of the request that
+# triggered the email follows.
+recovery-request-origin = This request came from:
+recovery-copy-paste = Copy and paste the url below within the next hour to create a new password.
```
Comment (dschom) [inline packages/fxa-auth-server/lib/senders/emails/templates/recovery/en.ftl:7]: nit: Should url be capitalized?

**Review by dschom — APPROVED** [review]
LGTM!

**Comment by chenba** [comment]
Made the suggested changes from @xlisachan in the Jira.

File: packages/fxa-auth-server/lib/senders/emails/templates/recovery/en.ftl [hunk 2]
```diff
@@ -1,4 +1,8 @@
 recovery-subject = Reset your password
-recovery-title = Need to reset your password?
-recovery-description = Click the button within the next hour to create a new password. The request came from the following device:
+recovery-title-2 = Forgot your password?
+# Information on the browser, IP address, date and time of the request that
+# triggered the email follows.
+recovery-request-origin = We received a request for a password change on your Firefox account from:
```
Comment (xlisachan) [inline packages/fxa-auth-server/lib/senders/emails/templates/recovery/en.ftl:5]: `We received a request for a password change on your { -product-firefox-account } from:`

**Review by xlisachan — APPROVED** [review]
r+wc 👍🏼

**Review by bcolsson — APPROVED** [review]

**Pull request merged**